src/commands.rs

```diff
@@ -1020,17 +1005,17 @@ pub fn pws_get(
   quiet: bool,
 ) -> anyhow::Result<()> {
   with_password_safe(ctx, |ctx, pws| {
-    check_slot(&pws, slot).context("Failed to access PWS slot")?;
+    let slot = pws.get_slot(slot).context("Failed to access PWS slot")?;
```

Comment:
Can you please check that we still report a proper user-understandable error in all three cases? I think we should be good, but doesn't hurt to double check.

Reply:
We now get the nitrokey-rs error variants:

```
$ nitrocli pws get 20
Library error: The given slot is invalid
$ nitrocli pws get 15
Command error: The given slot is not programmed
```

I think that’s fine.

Reply:
Though we lose the context when [downcasting](https://github.com/d-e-s-o/nitrocli/blob/211a25421c879097ec3a8b5e030795d5f84c7261/src/commands.rs#L277) the error.  As `downcast` consumes the original error, I’m not sure how to fix this.